src/sql/workbench/contrib/query/browser/actions.ts

```diff
@@ -67,6 +67,12 @@ export class SaveResultAction extends Action {
 	}
 
 	public async run(context: IGridActionContext): Promise<boolean> {
+		this.notificationService.notify({
+			severity: Severity.Info,
+			message: localize('jsonEncoding', "Results encoding will not be saved when exporting to JSON, remember to save with desired encoding once file is created."),
+			neverShowAgain: { id: 'ignoreJsonEncoding', scope: NeverShowAgainScope.WORKSPACE }
```

Comment:
I would say `GLOBAL` scope here would make more sense. SQL queries aren't really tied to workspaces so having this saved to a workspace would probably just be confusing